### ISSUE TITLE
telemetryLoggerMiddleware code issue for TS

### ISSUE THREAD

**Issue by rjgmail88** [microsoft/botframework-solutions#3385]
#### What project is affected?
Add a Skill in Power virtual agents
https://docs.microsoft.com/en-us/power-virtual-agents/configuration-add-skills#validation-performed-during-registering-a-skill

#### What language is this in?
Typescript

#### What happens?
When we try to executes steps mentioned in above document where are trying to add BF skill to PVA agent, following piece of code breaks 

File: /node_modules/botbuilder-core/lib/telemetryLoggerMiddleware.js
Function: fillReceiveEventProperties(...) has following lines 

```
properties[this.telemetryConstants.recipientIdProperty] = activity.recipient.id;
properties[this.telemetryConstants.recipientNameProperty] = activity.recipient.name;
```

As recipient is coming as NULL this is throwing an exception. So we had to modify the lines as follows which just sets recipient.id and recipient.name to empty string. 

```
properties[this.telemetryConstants.recipientIdProperty] = ( activity && activity.recipient && activity.recipient.id ) ? activity.recipient.id : '';

properties[this.telemetryConstants.recipientNameProperty] = ( activity && activity.recipient && activity.recipient.name ) ? activity.recipient.name : '';

```
#### What are the steps to reproduce this issue?
Steps 1: Create a BF TS skill (nodejs)
step 2: create a PVA bot (follow https://docs.microsoft.com/en-us/power-virtual-agents/fundamentals-get-started)
Step 3: Add a skill you created at step 1 to your PVA bot as step 2 using manifest file (follow https://docs.microsoft.com/en-us/power-virtual-agents/configuration-add-skills#validation-performed-during-registering-a-skill
)

#### What were you expecting to happen?
 We are expecting that recipient should be populated or this code check need to be added in. 

#### Can you share any logs, error output, etc.?
No

#### Any screenshots or additional context?

![image](https://user-images.githubusercontent.com/32375467/82361820-050a5800-99d1-11ea-8ed2-624a6738cd67.png)

**Comment by dfavretto** [comment]
Hi @rjgmail88, thanks for reporting this issue! Looking at the information you provided, this seems to be a problem in the `telemetryLoggerMiddleware` code, which is located in the SDK packages, more precisely the `botbuilder-core` package. I think this issue should be transferred to the [botbuilder-js](https://github.com/Microsoft/botbuilder-js) repository, is that okay with you?

**Comment by rjgmail88** [comment]
yes that is fine. Sorry I didn't log this to right place as I wasn't 100% sure on where to log.

**Comment by dfavretto** [comment]
No worries @rjgmail88 😊! @stevengum could you possibly transfer this issue?
